t/tearano.md

```diff
@@ -0,0 +1,13 @@
+# Tearano
+
+
+
+![Tearano](tearano.jpg)
+
+
+Tearano is a musical group from the southwestern region of Madagascar. They perform a contemporary form of the traditional beko genre, fusing southern vocal harmonies and tsapiky rhythms with modern instrumentation heavily featuring synthesizers and electric guitars.
+
+
+**Genre:** [Beko](https://app.slack.com/client/TSMBL8WEL/CSPHP6TGE),[Tsapiky](https://app.slack.com/client/TSMBL8WEL/CSPHP6TGE)
```

Comment:
?Why a link to slack?

Reply:
I fixed it now so please check and let me know if there is any mistake.

Thank you,